src/hooks/useBasisForm/validation/validateInput.ts

```diff
@@ -0,0 +1,23 @@
+const NUMERIC_REGEX = /^\d*$/;
+const DECIMAL_REGEX = /^\d*(\.\d{2})?$/;
```

Comment:
Can we consolidate these and the ones defined in `internal/InternalInput.tsx` so we have a single source of truth?